utils/responseUtil.ts

```diff
@@ -0,0 +1,56 @@
+import { NextApiResponse } from "next";
+
+/**
+ * @class ResponseUtil: Class for handling responses
+ */
+export class ResponseUtil {
+    /**
+     * Method for returning a status 200 success response
+     * @param res NextApiResponse
+     * @param message Optional message
+     * @param data Optional data
+     */
+    returnSuccess(
+        res: NextApiResponse,
+        message?: string,
+        data?: unknown,
```

Comment:
why do we have both a message and a data object? i think it would make more sense to just have the caller pass in the object that should be returned to the caller

Reply:
It was just to accommodate for the current places where we're returning a response. For mailing, the code was returning like `Mail successfully sent` or something. We can change to just be data

Reply:
done